src/main/java/marquez/api/models/DbTableRequest.java

```diff
@@ -15,19 +15,30 @@
 package marquez.api.models;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
 import javax.annotation.Nullable;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
+import marquez.common.models.DbColumn;
 
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public final class DbTableRequest extends DatasetRequest {
+  private final List<DbColumn> columns;
```

Comment:
Given our _offline_ discussion, let's define `List<DbColumn>` as `List<Field>` and push the list to up to `DatasetRequest`

Reply:
fixed.